static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -103,7 +103,11 @@ export const init = (): Promise<void> => {
         );
 
         onIabConsentNotification(state => {
-            const npaFlag = Object.values(state).includes(false);
+            // typeof state === 'boolean' means CCPA mode is on
+            const npaFlag =
```

Comment:
In case we're in CCPA mode, we need to leave the NPA flag alone. Enforcement of CCPA opt-out in Google is covered in a card for next week.

So in short, for this pull request:
- If CCPA is present , don't touch the NPA flag setting.
- To know if CCPA is present. I think it would be better to test for the presence of window.__uspapi

Reply:
👍 Had a look at the card. Seems simple enough to implement as part of this PR.